server/httpRequest.h

```diff
@@ -0,0 +1,32 @@
+#ifndef HTTPREQUEST_H
+#define HTTPREQUEST_H
+
+#include <string>
+#include <map>
+class HttpRequest{
+public:
+	bool Parse(const std::string request);
+	std::string getMessageBody();
+	std::string getMethod();
+	std::string getResourcePath();
+	std::string getVersion();
+	std::string getField(std::string header);
+
+	static const std::string GET;
```

Comment:
Use an ENUM instead of a string for switching values. An enum over types of words, which has a way to convert them into strings would be best (see: nginx-configparser)